composer.json

```diff
@@ -13,8 +13,8 @@
     },
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.1|^3.0",
-        "doesntmattr/mongodb-migrations": "^1.1",
+        "symfony/framework-bundle": "~2.1|^3.0|^4",
+        "doesntmattr/mongodb-migrations": "^1.1|dev-master",
```

Comment:
@w473 
`dev-master` makes me a little nervous, I think we would likely have a `1.2` with your `mongodb-migrations` changes ( https://github.com/doesntmattr/mongodb-migrations/pull/13 ) and `^1.1` will include it..